osidb/models.py

```diff
@@ -437,6 +438,7 @@ def allowed(self):
     @property
     def from_snippet(self):
         return {
+            self.CVEORG,
```

Comment:
`FlawSource` already contains `CVE` and `MITRE` values. I created new `CVEORG`, so we can differentiate collector flaws based on sources.